Detectors/HMPID/calibration/include/HMPIDCalibration/HMPIDDCSProcessor.h

```diff
@@ -177,6 +138,94 @@ class HMPIDDCSProcessor
     arNmean.clear();
   }
 
+  void checkEntries(const std::vector<TF1>& arQthresh,
```

Comment:
This method just prints a warning on detecting the default object, w/o any effect on the output, is this what you want?

Reply:
Yes, indeed this method is not really useful, I removed it.